packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -155,9 +155,8 @@ export default class CashierStore extends BaseStore {
             ObjectUtils.isEmptyObject(this.p2p_offer_list) &&
             !this.root_store.client.is_virtual
         ) {
-            const is_agent = !(await WS.p2pAgentInfo()).error;
-
-            if (is_agent || (await this.hasP2pOffer())) {
+            const agent_error = ObjectUtils.getPropertyValue(await WS.p2pAgentInfo(), ['error', 'code']);
+            if (!(agent_error === 'PermissionDenied')) {
```

Comment:
if we're going to call p2p to check for permissiondenied and don't care to check if client is agent or not anymore I would rather call p2p_offer_list instead, calling p2p_agent_info has no benefit for us. also unused functions and unused websocket calls for p2p should be removed in socket page if you're not calling them here anymore

Reply:
we will in notifications use websocket calls for p2p,. i thought that p2p_agent_info will have lesser response size when there is no error, meanwhile offer_list will be heavy.